app/src/main/java/com/dsu/poc/project/repository/login/LoginRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.poc.project.repository
+
+class LoginRepository {
+}
```

Comment:
project is not required. Please directly use the ui package. We know that the entire code is a part of a single project.

Reply:
changed.